deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,155 @@
+# OpenTelemetry Pivotal Cloud Foundry (PCF) Buildpack
+
+A [Cloud Foundry buildpack](https://docs.pivotal.io/application-service/2-11/buildpacks/) to install
+the OpenTelemetry Collector for use with PCF apps.  This will probably work for generic Cloud Foundry
```

Comment:
It may be clearer to tighten this compatibility a bit and not carry over the agent phrasing about generic functionality.

Reply:
Removed the extra sentence.